cli/azd/internal/scaffold/scaffold.go

```diff
@@ -149,6 +149,13 @@ func ExecInfraFs(
 		return nil, fmt.Errorf("scaffolding main.parameters.json: %w", err)
 	}
 
+	if spec.AiFoundryProject != nil {
+		err = executeToFS(fs, t, "ai-models.bicep", "ai-models.bicep", spec)
```

Comment:
Rename to `ai-project.bicep`?

Don't know how you feel about `ai/project.bicep` as a director vs. `ai-project.bicep` as a file (depends on how far we're building here).